tests/test_auth.py

```diff
@@ -24,7 +24,7 @@ def test_incorrect_password(db_session):
     db_session.add(employee)
     db_session.commit()
     error = login("vgv", "unknown")
-    assert (error == "Incorrect password.")
+    assert (error == "login.failed")
```

Comment:
@paulodamaso Was login the only place where we returned errors so far?

Reply:
@amihaiemil I think that they happen only on login.